playground/pages/Home.vue

```diff
@@ -11,12 +11,69 @@ const { data } = await storyblokApi.get("cdn/stories/vue", {
 
 const story = ref(null);
 story.value = data.story;
-
+console.log(story.value);
 onMounted(() => {
   useStoryblokBridge(story.value.id, (evStory) => (story.value = evStory));
 });
+
```

Comment:
Are we planning to use this commented code later in time?

Reply:
Is there for debugging purposes.